src/parsers/psmt2_to_alt_ergo.ml

```diff
@@ -401,8 +423,8 @@ module Translate = struct
     | Cmd_SetLogic _ -> not_supported "set-logic"; acc
     | Cmd_SetOption _ -> not_supported "set-option"; acc
     | Cmd_SetInfo _ -> not_supported "set-info"; acc
-    | Cmd_Push _ -> not_supported "push"; assert false
-    | Cmd_Pop _ -> not_supported "pop"; assert false
+    | Cmd_Push n -> (translate_push n (pos command)) @ acc
```

Comment:
let n : int = `nb_contexts n` to do:
- `int_of_string n`, or
- fail with the message in `must_not_happen`

Then, `translate_xxx` could be called with `n : int`

Reply:
Should be ok with previous comments and changes